packages/README.md

```diff
@@ -41,3 +41,25 @@ should be `kbn-datemath`.
 All new packages should use the `@kbn` namespace, and should be marked with
 `"private": true`.
 
+## Unit tests for a package
+
+Currently there are two patterns used to test packages, one using Mocha and one using Jest. These patterns emerged out of convention and we'd like to make them more similar to each other in the near future.
+
+### 1. Mocha
+Today a package can follow the pattern of having a `__tests__` directory in each source code directory of a package which contains the tests for that module. These are usually run by Mocha.
+
+If a package's tests should be run with Mocha, you'll have to opt-in to run them by appending the package's test file pattern(s) to Kibana's `tasks/config/simplemocha.js` file. These will then be run by the unit test runner.
+
+* `yarn test` or `./node_modules/.bin/grunt test` runs all unit tests, including Mocha tests.
```

Comment:
@kjbekkelund is this correct?

Reply:
yeah, i think this is correct.

Reply:
Yeah, looks correct.

Btw, a nice feature in Yarn is that anything within `./node_modules/.bin/` can be run directly by `yarn`, e.g. `yarn grunt test` instead of `./node_modules/.bin/grunt test`. And the one below this line can be `yarn grunt simplemocha:all`

Reply:
Oh sweet!